### ISSUE TITLE
Storybook 7.0

### ISSUE THREAD

**Issue by shilman** [storybookjs/storybook#13491]
We're accumulating a number of breaking or highly disruptive changes in the package ecosystem that might require a major release. Collecting them here to explore a 7.0 release or possibly a long-running prerelease that allows us to unblock users.

- [ ] Webpack@5 #9216 (50+ 👍)
- [ ] PostCSS@8 #12668 (40+ 👍)
- [ ] Emotion@11 #13145 (5+ 👍)
- [ ] Lit-html@2 #13489
- [ ] MDX@2 [tweet](https://twitter.com/4lpine/status/1340679424191033345)

**Comment by Diokuz** [comment]
Guys, why these are dependencies, not peer dependencies? At least, consider to ship another package for full-control-mode only – we dont need any defaults for webpack or postcss in our projects.

**Comment by Diokuz** [comment]
Actually, same problem was for React17, and we waited for weeks for 5.1 version of storybook.

Consider to use `Dependency Inversion` pattern, please.

**Comment by dartess** [comment]
Hey! I could not find information: will the `storybook@7` stop shipping with webpack 4 by default?

As far as I can tell packages `@storybook/addon-docs`, `@storybook/react` and even `@storybook/builder-webpack5` (through `@storybook/core-common`) now install webpack 4. This brings with it a lot of outdated packages that cause installation warnings.

**Comment by shilman** [comment]
@dartess yes, SB7 will probably ship with no builder by default and you'll need to choose between webpack4, webpack5, vite, or something else.

**Comment by BogdanMoraru82** [comment]
Will storybook@7 support pnpm package manager?

**Comment by mcmxcdev** [comment]
Is there any ETA for Storybook v7? We would really like to get rid of all the deprecated dependencies that get pulled in due to webpack4.

**Comment by dartess** [comment]
Is there a list of browsers that are supported in version 7? I heard you are planning to drop IE11. Do you have a target list of browsers and their versions that will be supported?

**Comment by IanVS** [comment]
@dartess here's a recent slide from a monthly roadmap meeting that discusses the breaking changes in 7.0 to browser and node support: https://docs.google.com/presentation/d/187z_4QySXzrPReEpIYhdKWgwQDWMpxSNI0ZXb1NVXUo/edit#slide=id.g14ffbccc1fa_0_33

TL;DR: Storybook will support "modern browsers" (Chrome 100+, not sure about the details on Safari, Firefox, etc.), and Node 14+.

**Comment by jonniebigodes** [comment]
Closing this for now, as the changes mentioned are already a part of the recent Storybook 7.0 release. 

@shilman let me know if I'm missing something and I'll gladly re-open it
